libs/schematics/project.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "schematics",
```

Comment:
We could maybe have a nicer name for this, e.g.

`@ngx-nuts-and-bolts/schematics`

Just to be consistent, because all other packages have this prefix.